apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs

```diff
@@ -217,7 +219,8 @@ public static SpannerCommandTextBuilder FromCommandText(string commandText)
         {
             GaxPreconditions.CheckNotNullOrEmpty(commandText, nameof(commandText));
             commandText = commandText.Trim();
-            var commandSections = commandText.Split(' ');
+            // Split(new char[0]) splits the string using all whitespace characters.
+            var commandSections = commandText.Split(new char[0], StringSplitOptions.RemoveEmptyEntries);
```

Comment:
The documentation suggests that we could use `(char[]) null` instead of `new char[0]`, which would avoid creating an empty array on every call.

Reply:
Good point, done.